README.md

```diff
@@ -6,9 +6,9 @@
 
 Deepcrawl Test is a platform for ensuring that deployments are not causing SEO regressions. It works by connecting DeepCrawl to your CI/CD pipeline to kick off a crawl of a staging or development environment, and returns a "passed" or "failed" status along with notifications about the specific regressions.
 
-Users can define specific tests, thresholds, and severity of regressions in the Automator app, as well as view test results or set notification preferences.
+Users can define specific tests, thresholds, and severity of regressions in the Automation Hub app, as well as view test results or set notification preferences.
 
-## Connecting Automator to your CI pipeline
+## Connecting Automation Hub to your CI pipeline
```

Comment:
@rthiiyer82 should we do `Deepcrawl Test` instead?

Reply:
I don't think so. We haven't changed to Deepcrawl Test officially and hence left it as is